docs/reference/project_package.md

```diff
@@ -0,0 +1,9 @@
+## Packaging data
```

Comment:
Rename this file to `package.md`? It would be consistent with `schema.md` (which isn't project_schema.md`).

Reply:
Renamed in https://github.com/open-contracting/infrastructure/pull/89/commits/c7df5e81b9bb577a9c184aeab4784b80864c3c05